src/settings/HRIDHandling/HRIDHandlingForm.js

```diff
@@ -38,13 +30,40 @@ const HRIDHandlingForm = ({
   handleSubmit,
   pristine,
   submitting,
-  reset,
-  mutator,
-  children,
+  render,
+  removeZeroes,
+  form: {
+    change,
+    reset,
+    getState,
+  },
 }) => {
-  const calloutRef = useRef();
-  const [isConfirmModalOpen, setConfirmModalOpen] = useState(false);
-  const [hridSettings, setHridSettings] = useState({});
+  const fieldsValues = getState().values;
+
+  useEffect(() => {
+    if (removeZeroes) {
```

Comment:
@mariia-aloshyna the `if` and  `else` blocks here look very similar. Could we clean it up a bit with moving the `if (removeZeroes) {` inside the `forEach` perhaps add `if (removeZeroes) {` inside `if (entityStartNumber) {`

Reply:
Thanks, @mkuklis ! Done.